test/TestApp/TestApp.cpp

```diff
@@ -44,16 +44,16 @@ namespace PlayFabUnit
         // Add PlayFab API tests.
         PlayFabApiTest pfApiTest;
         pfApiTest.SetTitleInfo(testInputs);
-        testRunner.Add(pfApiTest);
+        //testRunner.Add(pfApiTest);
```

Comment:
I do not approve of disabling the tests for this feature.
These tests must execute, test the QoS feature, and pass in order for this checkin to be accepted.

Reply:
Oops. This was disabled while debugging.